src/templates/category.html

```diff
@@ -26,9 +26,9 @@ <h1 class="cat-{{ category }} cat-icon">{{ category_name }}
   {% if false %}
       <a href="{{ search_url }}" class="view-all">{{ _('View All') }}</a>
   {% endif %}
-  <ol class="container app-list">
+  <ol class="container app-list {{ 'paginated' if response.meta.next }}">
```

Comment:
'ul' is also fine. For search results and new 'ol' makes some sense since they are ordered but we don't show the number anyway it's a bit of a moot point.